docs/CHANGELOG.asciidoc

```diff
@@ -13,11 +13,210 @@
 
 This section summarizes the changes in each release.
 
+* <<release-notes-7.0.0-beta1>>
 * <<release-notes-7.0.0-alpha2>>
 * <<release-notes-7.0.0-alpha1>>
 
 --
 
+[[release-notes-7.0.0-beta1]]
+== {kib} 7.0.0-beta1
+
+coming[7.0.0-beta1]
+
+[float]
+[[breaking-7.0.0-beta1]]
+=== Breaking changes
+
+For more details about breaking changes in this release, see 
+<<breaking-changes-7.0, Breaking changes in 7.0>>.
+
+Design::
+* Removes references to K6 EUI, using the default UI theme in its place {pull}29152[#29152]
+Geo::
+* Removes the deprecated `regionmap` and `tilemap` kibana.yml configurations. 
+Removes the backwards compatibility logic that checked for duplicative settings 
+`map.regionmap` and `map.tilemap`. {pull}28609[#28609]
+Monitoring::
+* Refactors index patterns from configurable to constants {pull}29528[#29528]
+Management::
+* Adds `configPrefix` to Index Management, License Management, Rollup Jobs, and 
+Upgrade Assistant {pull}30149[#30149]
+* Removes nested table splits from table visualization {pull}26057[#26057]
+* Standardizes use of `index_pattern` in UI code and URLs, replacing `indices` 
+and `index`  {pull}29151[#29151] and {pull}29481[#29481]
+* Removes app-specific dark mode options and replaces with a global dark mode uiSetting {pull}29017[#29017]
+Operations::
+* Defers the optimize step in the plugin installer to server start {pull}26983[#26983]
+Platform::
+* Removes deprecated `i18n.defaultLocale` setting {pull}29325[#29325]
+* Removes previously deprecated `elasticsearch*` settings. {pull}29496[#29496]
+* Adds new "references" attribute to saved objects for relationships {pull}28199[#28199]
+Reporting::
+* Removes Phantom dependency {pull}27142[#27142]
+Security::
+* Removes deprecated SSL settings {pull}28622[#28622]
+* Removes legacy fallback {pull}29107[#29107]
+Visualizations::
+* Removes Timelion from the app switcher by default {pull}30131[#30131]
+
+
+[float]
+[[deprecation-7.0.0-beta1]]
+=== Deprecations
+Machine Learning::
+* Updates REST endpoint calls to remove _xpack {pull}28069[#28069]
+Visualizations::
+* Removes old visualize wizard URL {pull}28982[#28982]
+* Removes legacy response handler {pull}28985[#28985]
+* Removes `legacyIcon` from visualizations {pull}28981[#28981]
+
+
+[float]
+[[enhancement-7.0.0-beta1]]
+=== Enhancements
+Kibana App::
+* Replaces Angular timepicker with EuiSuperDatePicker {pull}29204[#29204]
+* Upgrades Kibana to Elasticsearch 7.0 {pull}29184[#29184]
+* Visualizes renderer to support Canvas visualizations {pull}29170[#29170]
```

Comment:
Remove - we actually disabled this change for 7.0, so it should not be highlighted.